--- libvips-8.7.4/libvips/foreign/svgload.c.orig	2019-01-18 13:52:04.000000000 +0100
+++ libvips-8.7.4/libvips/foreign/svgload.c	2022-04-13 18:08:26.557185578 +0200
@@ -78,7 +78,7 @@
 /* Old librsvg versions don't include librsvg-features.h by default.
  * Newer versions deprecate direct inclusion.
  */
-#ifndef LIBRSVG_FEATURES_H
+#if !defined(LIBRSVG_FEATURES_H) && !defined(RSVG_FEATURES_H)
 #include <librsvg/librsvg-features.h>
 #endif
 
